tests/vcc/preamble.h

```diff
@@ -796,6 +808,21 @@ struct aws_event_loop *aws_event_loop_new_default(struct aws_allocator *alloc, a
         \fresh(c_mutex) && \wrapped0(c_mutex) && \claims_object(c_mutex, &(epoll_loop_of(\result)->task_pre_queue_mutex))))
 ;
 
+struct aws_event_loop *aws_event_loop_new_default_with_options(
+    struct aws_allocator *alloc,
+    const struct aws_event_loop_options *options
+    _(out \claim(c_mutex))
+)
+    _(requires \wrapped(alloc))
+    _(maintains \wrapped(options))
+    _(ensures \result == NULL ||
+       (\wrapped0(\result) &&
+        \fresh(\result) && \malloc_root(\result) &&
+        \fresh(epoll_loop_of(\result)) && \malloc_root(epoll_loop_of(\result)) &&
+        ownership_of_epoll_loop_objects(epoll_loop_of(\result)) &&
+        \fresh(c_mutex) && \wrapped0(c_mutex) && \claims_object(c_mutex, &(epoll_loop_of(\result)->task_pre_queue_mutex))))
```

Comment:
The VCC specification says, informally:
- Pre: The current thread must own the alloc and options objects
- Post: The result is either NULL or a fresh malloc'ed event loop owned by the current thread